webpack/helpers.js

```diff
@@ -0,0 +1,47 @@
+var path = require('path');
+var root = path.resolve(__dirname, '..');
+var webpack = require('webpack');
+
+exports.installVendorDLL = function(config, dllName) {
+  // DLL shizzle. Read more about this in /webpack/dlls/README.md
+  if (process.env.WEBPACK_DLLS === '1') {
```

Comment:
This if statement is redundant at least in this specific use case because the calling code does the same check